compiler/src/dotty/tools/dotc/typer/Checking.scala

```diff
@@ -156,7 +156,7 @@ object Checking {
     if (arg.tpe.widen.isRef(defn.NothingClass) ||
         !paramBounds.exists ||
         arg.tpe.hasSameKindAs(paramBounds.bounds.hi)) arg
-    else errorTree(arg, em"Type argument ${arg.tpe} has not the same kind as its bound $paramBounds")
+    else errorTree(arg, em"Type argument ${arg.tpe} doesn't have the same kind as its bound $paramBounds")
```

Comment:
I think we tend to avoid contraction in compiler messages:
```suggestion
    else errorTree(arg, em"Type argument ${arg.tpe} does not have the same kind as its bound $paramBounds")
```

Reply:
Maybe just emit some arrow `->` notation.

Reply:
well shoot, I didn't know y’all stood on ceremony round these parts, us shitkickers better class it up